.github/workflows/test.yml

```diff
@@ -53,6 +53,9 @@ jobs:
           - php: '5.6'
             wp: '5.5'
             experimental: false
+          - php: '5.6'
+            wp: '5.2'
+            experimental: false
```

Comment:
WP only dropped support for PHP 5.6 in WP 6.3, so I'd suggest setting the `wp` key here to `6.3` to compliment the build against WP 5.5 which is already in the matrix.

Reply:
I only added in because previously wp 5.2 in the matrix was also checking php 5.6. But since WP latest, it's php 7 and above so php 5.6 needs to be removed from the matrix.

Reply:
Nice suggestion on adding wp 6.3 in for the check, I've added it now.